pages/api/events-tags.ts

```diff
@@ -24,7 +24,7 @@ export default async function handler(
     aggs: {
       events_tags: {
         terms: {
-          field: 'field_event_tags',
+          field: 'field_event_tags.keyword',
```

Comment:
This currently works without the `keyword` part on test and production, so we can't add it back without breaking the event tags again.